### DISCUSSION TITLE
Working of Wix bal:Condition in bundle.wxs file

### DISCUSSION THREAD

**Discussion by AnjaliMamidi** [wixtoolset/issues#6919] [Questions]
Hello,

Can anyone provide insights on how bal:Condition works in bundle.wx file?
I've provided the condition but bal:Condition is not getting evaluated.
Any suggestion or help would be appreciated.

Thanks.

**Comment by barnson** [comment]
WiX doesn't use issues for support. We moved this issue to the [experimental GitHub discussions area][gh]. You might also consider asking your question on the [wix-users mailing list][ml] or posting on [Stack Overflow][so].

[gh]: https://github.com/wixtoolset/issues/discussions?discussions_q=category%3AQuestions
[ml]: http://wixtoolset.org/documentation/mailinglist/
[so]: https://stackoverflow.com/questions/tagged/wix

**Comment by robmen** [comment]
The `bal:Condition` is evaluated by the wixstdba. You can [see the code here](https://github.com/wixtoolset/wix4/blob/d5985a1688bc878e42ffd3ce3939fa52303cab16/src/ext/Bal/wixstdba/WixStandardBootstrapperApplication.cpp#L4046)
